packages/client/src/pages/Topic/TopicMessage.tsx

```diff
@@ -1,8 +1,14 @@
-import { Avatar, Box, Typography } from "@mui/material";
-import { IMessages } from "@/store/forumStore";
+import { Avatar, Box, Typography, Button } from "@mui/material";
+import { ThumbUp, ThumbDown } from "@mui/icons-material";
+import { IMessages, useTopicStore } from "@/store/forumStore";
+import { useUserStore } from "@/store";
 
 export function TopicMessage(props: IMessages) {
   const { date, topicId, message, likes, ownerId, dislikes } = props;
+  const messageId = props.id;
+  const likeMessage = useTopicStore((store) => store.likeMessage);
+  const dislikeMessage = useTopicStore((store) => store.dislikeMessage);
+  const { id } = useUserStore((store) => store.user);
```

Comment:
я не устаю варнить, что не стоит использовать деструктурирующее присваивание при работе с хуком селектора стейт менеджера (`useSelector`). При использовании подобного подхода, компонент будет ререндериться каждый раз при изменении любого поля объекта `store.user`. Даже тех полей, которые не присваиваются переменным. есть два выхода: использовать в качестве второго аргумента хуку - функцию компаратор (которая будет определять случаи, когда необходим ререндер) или возвращать из хука только те поля, которые необходимы. Я - приверженец использования второго подхода, но @kolosochek полюбил первый. Так или иначе, нужно сделать выбор в пользу одного из предложенных методов =-)
```suggestion
  const id = useUserStore((store) => store.user.id);
```